src/backend/distributed/deparser/deparse_role_stmts.c

```diff
@@ -0,0 +1,139 @@
+/*
+ *
+ * static void AppendAlterRoleStmt(StringInfo buf, AlterRoleStmt *stmt);
+ *
+ */
+
+#include "postgres.h"
+
+#include "lib/stringinfo.h"
+#include "nodes/parsenodes.h"
+#include "utils/builtins.h"
+#include "distributed/citus_ruleutils.h"
+
+static void AppendAlterRoleStmt(StringInfo buf, AlterRoleStmt *stmt);
+const char * DeparseAlterRoleStmt(AlterRoleStmt *stmt);
```

Comment:
As I noted in other place, we typically do not declare external/public functions here as well.